cmd/mkmanifest/main.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// This is a simple helper utility to create a '.syso' file that contains a
```

Comment:
// The mkmanifest command is a ...

or

// The mkmanifest tool is a ...

etc